pkgs/tools/misc/psw/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "psw";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "Wulfsta";
+    repo = pname;
+    rev = version;
+    sha256 = "10raj4899i01f5v13w0wxdnjjicql2wjblkq1zcagrfv3ly3d0fy";
+  };
+
+  cargoSha256 = "1w18rym0xnjk7vhrb2dc4cvhg659zbq5d2153gw2snxcbs7gh7r1";
+
+  meta = with stdenv.lib; {
+    description = "A command line tool to write random bytes to stdout";
+    homepage = "https://github.com/Wulfsta/psw";
+    license = with licenses; [ asl20 /* or */ mit ];
+    maintainers = with maintainers; [ wulfsta ];
```

You're missing a `platforms` designation here. Without it, Hydra won't build it and there won't be any binary substitutes available.

I would suggest the following:

```suggestion
    maintainers = with maintainers; [ wulfsta ];
    platforms = platforms.linux;
```

unless you can verify it works on darwin (in which case you can probably change it to `unix`).

---

It should work on Darwin, but I don't have the hardware to test. Added for Linux until that can be done.